flocker_bb/builders/flocker.py

```diff
@@ -573,6 +573,21 @@ def makeHomebrewRecipeTestFactory():
 
 def getBuilders(slavenames):
     builders = [
+        BuilderConfig(
+            name='flocker/functional/storage-driver',
+            builddir='flocker',
+            slavenames=slavenames[
+                'centos-7',
+                'ubuntu-14.04',
+                'flocker/functional/rackspace/centos-7/storage-driver'
+            ],
```

Comment:
We want to test our OpenStack backend on Rackspace on CentOS 7 and on Ubuntu 14.04 (because we're hoping to claim those all as supported platforms/configurations).  So we need to run the tests on CentOS 7 on Rackspace and on Ubuntu 14.04 on Rackspace.  Ditto for those two OS on AWS.

This can't be one builder because you can't be on **both** Rackspace and AWS at the same time and you can't be on **both** CentOS 7 and Ubuntu 14.04 at the same time.  So (Rackspace, AWS) x (CentOS, Ubuntu) is four builders.  The builders all sorta do the same thing - run the functional tests - but they are running them in different environments so getting different test coverage.